cmd/tetragon/main.go

```diff
@@ -321,9 +321,26 @@ func tetragonExecute() error {
 		}
 	}
 
+	// Explicitly log status every 1 hour
+	go logStatus(ctx, obs, 1*time.Hour)
```

Comment:
Do we need to make this configurable? Is so, let's split the two patches and do two different PRs.

Reply:
+1 on making this configurable

Reply:
Ok will split it in separate patch and think about making it configurable, I'm not a fan of adding too much options better try to do the right thing by default